templates/tease-search.twig

```diff
@@ -121,7 +121,7 @@
 							data-ga-category="Search Results"
 							data-ga-action="Call to Action"
 							data-ga-label="{{ ga_page_type }}">
-								{{ link_text ?: __( 'take action', 'planet4-master-theme' ) }}
+								{{ link_text ?: __( 'Take action', 'planet4-master-theme' ) }}
```

Comment:
If we change the case here, these won't be translated anymore, since the translation files still have the old case.
https://github.com/greenpeace/planet4-master-theme/blob/master/languages/planet4-master-theme-nl_NL.po#L305-L306

Reply:
Yep, I know. I guess we need to update the translations files then? Not sure how that works 🤔

Reply:
Indeed, you could include those changes in this PR. I guess this shouldn't cause issues for the loco plugin, but maybe we should double check this to be sure? cc @comzeradd

Reply:
We probably also need to update all translations too, as they all seem to be completely in lowercase (inc first letter) too. https://github.com/search?p=8&q=org%3Agreenpeace+%22msgid+take+action%22&type=Code

Reply:
I can add a commit here to update the translation files

Reply:
I set up the test instance with some languages so we can test others than just English. Also added WPML https://github.com/greenpeace/planet4-test-jupiter/blob/develop/composer-local.json

Reply:
I pushed a new commit with translation files updated. @mleray you have to pull it locally too, if you plan in making any further changes to this branch

Reply:
Hi @Inwerpsel - You could also [sync data](https://app.gitbook.com/@greenpeace/s/planet4/ci-cd/test-environments#data-synchronization) from NRO site.

Reply:
We also released these changes in all dev instances, so different languages can be tested these 🙂